cddl_gen/cddl_gen.py

```diff
@@ -2299,13 +2303,16 @@ def parse_args():
     code_parser.add_argument(
         "-t", "--entry-types", required=True, type=str, nargs="+",
         help="Names of the types which should have their xcode functions exposed.")
-    code_parser.add_argument(
-        "--time-header", required=False, action="store_true", default=False,
-        help="Put the current time in a comment in the generated files.")
     code_parser.add_argument(
         "-d", "--decode", required=False, action="store_true", default=False, help="Generate decoding code.")
     code_parser.add_argument(
         "-e", "--encode", required=False, action="store_true", default=False, help="Generate encoding code.")
+    code_parser.add_argument(
+        "--time-header", required=False, action="store_true", default=False,
+        help="Put the current time in a comment in the generated files.")
+    code_parser.add_argument(
+        "--git-sha-header", required=False, action="store_true", default=False,
```

Comment:
niffti: (nit: Feel Free To Ignore) Sticking with the `common part first`, should these be called `--header-time` and `--header-git-sha` to better highlight that they are related?

Reply:
I think I'll keep it as is, since --time-header already exists so I don't want to change the interface.